setup.py

```diff
@@ -98,16 +99,13 @@
     'mosaicml[tensorboard]>=0.20.1,<0.21',
 ]
 
-extra_deps['gpu'] = [
-    'flash-attn==1.0.9',
-    # PyPI does not support direct dependencies, so we remove this line before uploading from PyPI
-    'xentropy-cuda-lib@git+https://github.com/HazyResearch/flash-attention.git@v1.0.9#subdirectory=csrc/xentropy',
-]
-
 extra_deps['gpu-flash2'] = [
     'flash-attn==2.5.0',
 ]
 
+# Removing support for flash attention 1 without breaking the installation commands
```

Comment:
This should instead be a comment on flash2 saying here for backwards compatibility reasons imo.

Reply:
fair